Foundation/URLSession/URLSession.swift

```diff
@@ -201,8 +201,19 @@ open class URLSession : NSObject {
      * The shared session uses the currently set global URLCache,
      * HTTPCookieStorage and URLCredential.Storage objects.
      */
-    open class var shared: URLSession { NSUnimplemented() }
-    
+    open class var shared: URLSession {
+        return _shared
+    }
+
+    fileprivate static let _shared: URLSession = {
```

Comment:
Lazy?

Reply:
IIRC, all global stored variables (including all static stored variables) that are initialized with a block call are initialized lazily. `lazy` cannot be applied to them.

Reply:
Yeah, static properties are implicitly lazy.